contracts/cobalt/LoanEngine.sol

```diff
@@ -353,7 +353,17 @@ contract LoanEngine is Ownable, ERC721Base {
         require(loan.approved, "The loan is not approved by the borrower");
         require(loan.status == Status.request, "The loan is not a request");
         require(now < loan.requestExpiration, "Request is expired");
-        uint256 requiredTransfer = convertRate(loan.oracle, loan.currency, oracleData, loan.amount);
+        uint256 requiredTransfer;
+
+        if (loan.oracle == address(0)) {
+            requiredTransfer = loan.amount;
+        } else {
+            uint rate;
+            uint decimals;
```

Comment:
The rest of the contract uses uint256 and not the alias uint, also it can could be replaced by 
`(uint256 rate, uint256 decimals) = ...`